npm/vite-dev-server/client/initCypressTests.js

```diff
@@ -13,13 +13,13 @@ const supportFile = CypressInstance.config('supportFile')
 const projectRoot = CypressInstance.config('projectRoot')
 const devServerPublicPathRoute = CypressInstance.config('devServerPublicPathRoute')
 
-let supportRelativeToProjectRoot = supportFile.replace(projectRoot, '')
+if (supportFile) {
+  let supportRelativeToProjectRoot = supportFile.replace(projectRoot, '')
 
-if (CypressInstance.config('platform') === 'win32') {
-  supportRelativeToProjectRoot = supportFile.replace(projectRoot.replaceAll('/', '\\'))
-}
+  if (CypressInstance.config('platform') === 'win32') {
+    supportRelativeToProjectRoot = supportFile.replace(projectRoot.replaceAll('/', '\\'), '')
```

Comment:
Took me a while to figure out what was going on. Should we split this?

```js
const platformProjectRoot = projectRoot.replaceAll('/', '\\')
supportRelativeToProjectRoot = supportFile.replace(platformProjectRoot, '')
```

Reply:
I like it, addressed in [34c87f0](https://github.com/cypress-io/cypress/pull/21156/commits/34c87f021c40ec1a27067c534c0609aaecd7369e)